lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1386,6 +1536,24 @@ def test_team_count(self):
             else:
                 self.assertEqual(topic['team_count'], 0)
 
+    @ddt.unpack
+    @ddt.data(
+        ('student_enrolled', True),
+        ('student_on_team_1_private_set_1', True),
+        ('student_on_team_2_private_set_1', True),
+        ('staff', True)
+    )
+    def test_teamset_type(self, requesting_user, expect_see_private_teamsets):
+        topics = self.get_topics_list(
+            data={'course_id': self.test_course_1.id},
+            user=requesting_user
+        )
+        private_teamsets_returned = [
+            topic['name'] for topic in topics['results'] if topic['type'] == 'private_managed'
+        ]
+        expected_private_teamsets = 2 if expect_see_private_teamsets else 0
```

Comment:
Why not pass the expected length as a parameter instead of the boolean expect_see_private_teamsets?